pkg/cosign/remote/remote.go

```diff
@@ -108,7 +103,7 @@ LayerLoop:
 }
 
 type UploadOpts struct {
-	DupeDetector DupeDetector
+	DupeDetector mutate.DupeDetector
 	RemoteOpts   []remote.Option
```

Comment:
I know this was already here, but let's also standardize on `clientOpts` or `registryClientOpts` when we come across these to improve descriptiveness

Reply:
I can follow up to clean some of this stuff up